docs/terminal/sydney.md

```diff
@@ -335,15 +344,43 @@ It is vital that Approach controllers ensure all arriving aircraft are establish
 Aircraft are permitted to cross the MARLN corridor at or above `A060` without coordination with APP. DEP is responsible for separation with respect to aircraft in the corridor.
 
 #### SDN / SFW/SAN
-If aircraft are cleared off runway 11 into CTA, coordination is required from SDN (who will be receiving the aircraft from BK ADC) with SFW or SAN (depending on YSSY runway config) as the aircraft will pass closer than 1.5nm from the sector boundary.  
-Refer to [Sydney TCU Airspace Division](../../terminal/sydney/#airspace-division) for more information.
+If aircraft are cleared off runway 11 at YSBK into CTA, coordination is required from SDN (who will be receiving the aircraft from BK ADC) with SFW or SAN (depending on YSSY runway config) as the aircraft will pass closer than 1.5nm from the sector boundary.  
+Refer to [Sydney TCU Airspace Division](#airspace-division) for more information.
 
 !!! Example
     <span class="hotline">**SDN** -> **SFW**</span>: “Request left turn out of Bankstown”  
     <span class="hotline">**SFW** -> **SDN**</span>: "Approved" *(no callsigns need be used here)*  
+
+#### SRI / APP/DEP/DIR
+Heads-up coordination is not required from SRI to any other SY TCU position for aircraft:
+
+**Entering CTA *vertically*:**
+
+- Assigned the BCTA as the CFL; and
+- Requested level written in label as per [Annotations format](../../controller-skills/annotations/#requests-requirements-and-restrictions); and
+- Handed off to appropriate SY TCU controller
+
+**Entering CTA *laterally*:**
+
+- Assigned the BCTA or lower as the CFL; and
+- Handed off to appropriate SY TCU controller prior to **10nm** from lateral CTA boundary
+<br>
+<br>
+
+Heads-up coordination is not required from a SY TCU position to SRI for aircraft:
+
+**Leaving CTA *vertically*:**
+
+- Assigned 500ft above the BCTA as the CFL; and
+- Handed off to SRI
+
+**Leaving CTA *laterally*:**
+
+- Handed off to SRI upon termination of control services
+
```

Comment:
To clarify this procedure, are we saying that an aircraft requesting a harbour scenic (as an example) would not be coordinated between SRI and SDN, and instead would just be flashed to SDN and transferred to their frequency for clearance after identification?

At the risk of creating a coordination nightmare for ourselves, I wonder whether we should require coordination for all aircraft entering/leaving CTA and being transferred between APP/DEP and SRI.  I'm just thinking about the number of people who may try to issue clearance themselves as SRI or not consider traffic OCTA when issuing clearance to leave descending and create conflict unknowingly.

If we made the procedure something like:
1. OCTA aircraft calls SRI for airways clearance
2. SRI activates/creates FDR, issues squawk, identifies aircraft
3. SRI coords with applicable sector to inform of request.  Receiving sector advises whether clearance is available or not.
4. SRI hands aircraft to receiving sector who issues clearance.  If no clearance available, SRI informs aircraft and terminates identification.

Reply:
SME has advised that all aircraft going between SRI and CTA are heads-up coordinated IRL. Obviously would be good to reduce heads-up coord wherever possible on vatsim, but yeah the harbour scenic example is a fair point.

Maybe this is a little to vague for vatsim purposes, but something like: 

no coord required for all of the conditions already listed, AND their intentions must be clearly listed in the FDR (ie, routing and requested level is correct, HS1 written in label for harbour scenic request for example, "NC?" written in label for aircraft requesting DCT Camden NC, "NC" written in label for aircraft tracking DCT Camden NC

The clearance not being available due traffic/workload could be handled:

At best, told to SRI by the relevant SY TCU controller when they see the blue track requesting to fuck around at 5000 in the middle of Milk run.
At worst, when SRI flashes the handoff to them and they can say "nah fuck off"


Next level down would be perhaps to have SRI heads-up coordinate all aircraft to SY TCU, but keep it Voiceless from SY TCU to SRI with the conditions already listed. 

Thoughts?

Reply:
Yeah your last suggestion could work.  I'm hesitant to rely on annotations because I know that they'll be used by 30% of people and the other 70% will cause absolute havoc.  Even normal enroute sequencing stuff is all over the place (despite the detail the SOPs go into).

I'd support everything heads up coorded from SRI to TCU, voiceless from TCU to SRI with 000 in the CFL?

Reply:
> Yeah your last suggestion could work.  I'm hesitant to rely on annotations because I know that they'll be used by 30% of people and the other 70% will cause absolute havoc.  Even normal enroute sequencing stuff is all over the place (despite the detail the SOPs go into).
> 
> I'd support everything heads up coorded from SRI to TCU, voiceless from TCU to SRI with 000 in the CFL?

Agreed.

It'll be Voiceless from TCU to SRI with the CFL set to 500ft above BCTA, SRI will give the clearance to leave and traffic statement.

Reply:
Ok sounds good